docs/release/trg-2/trg-2-5.md

```diff
@@ -10,32 +10,35 @@ title: TRG 2.05 - Repository metafile
 
 ## Why
 
-Due to many products having more than one repository, we need a way of identifying dependent repositories automatically. This allows us to programmatically parse and analyze repositories for quality checks and for generating documentation across repositories.
+Due to many products having more than one repository, we need a way of identifying dependent repositories automatically. This allows us to analyze repositories for quality checks and for generating documentation across repositories.
 
 ## Description
 
-Having a `.tractusx` metafile helps individuals, release management, test management and others to determine basic information about a repository in the scope of **Eclipse Tractus-X**.
+To enable automatic analyzing of repositories, all repositories **MUST** have a file named `.tractusx` present on root-level.
```

Comment:
Perhaps "automatic analysis" might be more correct here.

Reply:
Yep you are right. Fixed